.github/workflows/ci_ec2_reusable.yml

```diff
@@ -123,7 +123,7 @@ jobs:
         id: preprocess
         shell: bash
         run: |
-          echo "nix-shell=${{ inputs.cbmc && 'ci-cbmc' || 'ci' }}${{ (inputs.compile_mode == 'cross' || inputs.compile_mode == 'all') && '-cross' || '' }}" >> $GITHUB_OUTPUT
+          echo "nix-shell=${{ (inputs.cbmc || inputs.compile_mode == 'cross' || inputs.compile_mode == 'all') && 'ci-cross' || 'ci' }}" >> $GITHUB_OUTPUT
```

Comment:
This switches the shell to ci-cross for cbmc? The commit message does not reflect that.